src/libraries/System.Private.CoreLib/src/System/Threading/Thread.cs

```diff
@@ -131,6 +131,7 @@ public CultureInfo CurrentUICulture
 
         public static IPrincipal? CurrentPrincipal
         {
+            [UnsupportedOSPlatform("browser")]
```

Comment:
I think this is a false positive from the code in `AppDomain.CurrentDomain.GetThreadPrincipal()`